packages/block-library/src/separator/edit.js

```diff
@@ -27,9 +28,23 @@ function SeparatorEdit( { color, setColor, className } ) {
 				style={ {
 					backgroundColor: color.color,
 					color: color.color,
+					width: width + '%',
 				} }
 			/>
 			<InspectorControls>
+				<PanelBody title={ __( 'Separator Settings' ) }>
+					<RangeControl
+						label={ __( 'Percentage width' ) }
```

Comment:
Range control has a initialPosition property https://github.com/WordPress/gutenberg//blob/1e40b281d1db725e512e383b5e00f3b59e333d5a/packages/components/src/range-control/README.md. when the value is unset the property allows the range to be in a give position e.g: 25, but the input is empty to allow the user to differentiate between the state the user set the width to 25 (inline style added) the user did not set any width but currently the default is 25 (no inline style is added).
Would it make sense to use this mechanism here?